### PR TITLE
Add uwsgi run option

### PR DESCRIPTION
**Pull request by ajmaddaford**
### What is the context of this PR?
Adds the option to run under uWSGI. There are three options within this - plain uWSGI, threaded or asynchronous (via gevent).

This branch has been used to run the tests in https://github.com/ONSdigital/eq-survey-runner-benchmark/pull/55.

### How to review 
- Run the app locally with the Makefile commands `run-uwsgi`, `run-uwsgi-threads` or `run-uwsgi-async`
- Check that `run-gunicorn` works as it did previously
- Run under Docker

### CODE REVIEW

File: run_app.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euo pipefail
```
Comment (MebinAbraham) [inline run_app.sh:3]: Will need to set `EQ_NEW_RELIC_ENABLED=False` in `.development.env`

**Comment by iwootten** [comment]
I can run by setting the EQ_NEW_RELIC_ENABLED=False as mentioned. Runner under docker (also on master) also seems to fail `Exception: Setting 'EQ_SUBMITTED_RESPONSES_TABLE_NAME' Missing`.

**Comment by ajmaddaford** [comment]
@iwootten That error looks like docker isn't being passed the required environment variables from the `.development.env` file. I ran using the `docker-compose.yml` file and it worked.

File: uwsgi.ini [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[uwsgi]
+http = :5000
+module = application:application
+buffer-size = 32768
```
Comment (MebinAbraham) [inline uwsgi.ini:4]: Was 32k a guesstimate or do we know our max request size?

Comment (ajmaddaford) [reply uwsgi.ini:4]: Our payloads vary between 6-10k, so it was a guesstimate based on that with headroom.

**Pull request merged**